cmd/kubectl-kcpforedgesyncer/cliplugins/workload/cmd/cmd.go

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
Would it be better to put all this `/cliplugins/` stuff in `pkg/cliplugins/`, like is done in kcp and following the usual pattern?